ui/src/components/Exchange/TokenSelect.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Form } from 'semantic-ui-react'
+
+import { TokenInfo } from './Exchange'
```

Comment:
maybe we should consolidate these interface types like we do in `Server.tsx`

Reply:
I consolidated these (along with the Tuple wrappers) into one file: `common/damlTypes.ts`